src/main/webapp/ProfilePage.html

```diff
@@ -44,15 +45,17 @@ <h6 class="card-subtitle mb-2" id="username"></h6>
           Edit Profile
         </button>
 
-        <div>
+        <div class="mt-3">
           <h2 class="text-center">Favorite Items</h2>
           <div
-            class="card"
             style="width: 45em; margin: 0 auto;"
             style="height: 10rem;"
           >
             <div id="item-container"></div>
           </div>
+          <h2 class="text-center mt-3">Recommended Users</h2>
```

Comment:
should we hide this text if there are no recommended users?

Reply:
I decided to keep the text, but I added dynamic text that tells the user to like more items if this section is empty.

Reply:
nice I like that idea :)